README.md

```diff
@@ -42,6 +42,18 @@ This script can be modified, to adjust how automate transfers works.  The full s
 * `--files`: If set, start transfers from files as well as folders.
 * `--hide`: If set, hides the Transfer and SIP once completed.
 
+Configuration - Getting Correct UUIDs
+-------------------------------------
+
+The automation script relies on installation-specific UUIDs. To obtain the transfer source UUID for script invocation, visit the 'Transfer Source' tab in the Archivematica Storage Space web dashboard. If a row is marked as a transfer souce its UUID value will be valid for a transfer. To fully automate AIP storage transfer, the processingMCP.xml file used in your transfer must be changed. For example, if you are using the defaultProcessingMCP.xml file provided by this tool in the pre-transfer directory, you need to modify the following section:
+
+    <preconfiguredChoice>
+      <appliesTo>b320ce81-9982-408a-9502-097d0daa48fa</appliesTo>
+      <goToChain>/api/v2/location/99d0466e-173d-4b67-98a3-490198e56925/</goToChain>
+    </preconfiguredChoice>
+
+The UUID after /location/ should correspond to a UUID of an AIP Storage location, also findable in the 'Transfer Source' tab in the Archivematica Storage Space web dashboard. Non-Archivematic Storage Space AIP storage destinations will likely need more configuration.
```

Comment:
Archivematica's normal workflow doesn't support storing AIPs outside the Storage Service. Can you change this to `Non-Archivematic Storage Space AIP storage destinations are unsupported.` or remove it?